.env_template

```diff
@@ -0,0 +1,13 @@
+DATABASE_URL=postgres://db:db@db/db
+PG_FEATURESERV_SYSTEM_PORT=9000
```

Comment:
for the moment we can just hardcode the port. currently i do not see a point making this configurable

Reply:
You sure? This is the system port, where it will be available on the host system. It could be the case that you have something else running on port 9000, or in the future you want to compare different builds by running them on different ports

Reply:
In my latest commit I did not resolve this yet, if you use the template .env file then it is as good as hardcoded anyway. But if you really insist then I will hard code it directly.

Reply:
it actually does not matter, we can leave it as is